### PR TITLE
clean root folder, added multiple choice

### PR DESCRIPTION
**Pull request by JulienDemarque**

### CODE REVIEW

File: web/src/pages/Question.js [hunk 1]
```diff
@@ -4,6 +4,13 @@ import Prism from 'prismjs';
 import '../prism.css';
 
 class QuestionPage extends Component {
+  constructor(props) {
```
Comment (treyhuffine) [inline web/src/pages/Question.js]: Just my personal opinion, but I prefer not to use constructors if possible. Instead I would do

```
state = {
  result: '',
}

validate = (e) => {
  ....
```

By making the validate an arrow function inside the class, it automatically binds the `this` to it

File: web/src/pages/Question.js [hunk 2]
```diff
@@ -12,6 +19,22 @@ class QuestionPage extends Component {
     Prism.highlightAll();
   }
 
+  validate(e) {
+    e.preventDefault();
+    //We could use querySelector on a reference to the form element instead of document...
+    if (document.querySelector('input[name="answer"]:checked')) {
+      const userAnswerIndex = document.querySelector('input[name="answer"]:checked').value;
+      const isCorrect = this.props.question.answers[userAnswerIndex].isCorrect;
+      if (isCorrect) {
```
Comment (treyhuffine) [inline web/src/pages/Question.js]:
```
// Add this outside the component
questionResults = {
  pending: 'Select an answer!',
  correct: 'Right Answer!',
  wrong: 'Wrong Answer... try again!',
};
// ...other code

const result = isCorrect ? questionResults.correct   : questionResults.wrong;
```

Comment (treyhuffine) [reply web/src/pages/Question.js]: Let me know if this doesn't make sense

File: web/src/pages/Question.js (hunk 1 above)
Comment (JulienDemarque) [reply web/src/pages/Question.js]: I needed the constructor in the end...

**Comment by adrianwix** [comment]
I will take a look later, you can merge if it works, and we can improve it later

File: web/src/pages/Question.js [hunk 3]
```diff
@@ -21,11 +63,23 @@ class QuestionPage extends Component {
...
           <code className="language-javascript">{file}</code>
         </pre>
-        <ul>
+        <form action="">
           {question.answers.map((a, index) => (
-            <li key={index}>{a.answer}</li>
+            <div key={index}>
+              <input
+                onChange={this.handleInputChange}
+                checked={this.state.inputsChecked[index] ? true : false}
```
Comment (treyhuffine) [inline web/src/pages/Question.js]: I think you should be able to just do `checked={this.state.inputsChecked[index]}`

Comment (JulienDemarque) [reply web/src/pages/Question.js]: I tried, it gave me an error that got fixed with the ternary operator... I didn't dig into why it happened though...

**Comment by JulienDemarque** [comment]
multiple choice logic

File: web/src/pages/Question.js (hunk 3 above)
Comment (adrianwix) [reply web/src/pages/Question.js]: True

**Pull request merged**